tests/test_main.py

```diff
@@ -525,8 +529,26 @@ def check_filename(self, filename, check_names):
                     correct = len(values) == 1
 
                 if not correct:
-                    print(source.text, '---', node_string(source, node), node.lineno,
-                          len(values), correct, values, file=sys.stderr, sep='\n')
+                    print(source.text, file=sys.stderr)
+                    print("---", file=sys.stderr)
+                    print("node without associated Bytecode", file=sys.stderr)
+
+                    print("node", ast.dump(node, indent=4), file=sys.stderr)
```

Comment:
Please keep this compatible with older versions of Python that don't have the `indent` argument. Same with printing `node.position` below. But this is good, thanks for making these failures clearer.

Reply:
👍